service/domain/graph/graph.go

```diff
@@ -4,79 +4,14 @@ import (
 	"sort"
 
 	"github.com/boreq/errors"
-	"github.com/planetary-social/scuttlego/internal"
-	"github.com/planetary-social/scuttlego/service/domain/feeds"
 	"github.com/planetary-social/scuttlego/service/domain/refs"
 )
 
-type Storage interface {
-	GetContacts(node refs.Identity) ([]*feeds.Contact, error)
-}
-
 type SocialGraph struct {
```

Comment:
Doesn't have a constructor now, kinda weird, will rethink in the future I guess.

Reply:
As long as it's built with the builder I don't think it makes any problem.